api/AudioBuffer.json

```diff
@@ -0,0 +1,659 @@
+{
+  "api": {
+    "AudioBuffer": {
```

Comment:
I think we should have an entry for AudioBuffer itself that is supported since 14 (prefixed).

Reply:
Also mdn_url can link to the AudioBuffer page from there.